### PR TITLE
Delete channels workflow

### PR DESCRIPTION
**Pull request by HassanTeslim007**
Workflow For Deleting channels completed
Screenshot_20210922-142137
Screenshot_20210922-142146
Screenshot_20210922-142155
Workflow For Deleting channels completed

### CODE REVIEW

**Comment by Blazebrain** [comment]
Fix Conflicts 

Also, does it return to the homepage after?

Once a channel is deleted, we should be directed to the homepage

**Comment by HassanTeslim007** [comment]
Conflicts Resolved and Yes, It returns to Homepage

**Comment by Blazebrain** [comment]
Can you refix boss?

Still has some

**Comment by HassanTeslim007** [comment]
Fixed boss... again

**Comment by HassanTeslim007** [comment]
Finally, Fixed again.

**Comment by YenumJay** [comment]
Pull latest changes from dev and resolve conflicts then you should be good.

**Comment by HassanTeslim007** [comment]
Done...

File: lib/ui/nav_pages/home_page/home_page_viewmodel.dart [hunk 1]
```diff
@@ -102,6 +135,17 @@ class HomePageViewModel extends StreamViewModel {
     return connectionStatus;
   }
 
+<<<<<<< HEAD
```
Comment (jeremiahseun) [inline lib/ui/nav_pages/home_page/home_page_viewmodel.dart:138]: This line should not be there

File: lib/ui/nav_pages/home_page/home_page_viewmodel.dart [hunk 2]
```diff
@@ -102,6 +135,17 @@ class HomePageViewModel extends StreamViewModel {
...
+  void navigateToJumpToScreen() {
+    _navigationService.navigateTo(Routes.dmJumpToView);
+  }
+
+  void navigateToStartDMScreen() {
+    _navigationService.navigateTo(Routes.startDmView);
+  }
+
+=======
+>>>>>>> d4f077f1ecab6b315ee58c05d2e2116f60d012a3
```
Comment (jeremiahseun) [inline lib/ui/nav_pages/home_page/home_page_viewmodel.dart:148]: This also
And line 147

File: lib/ui/shared/dialogs/delete_channels_dialog.dart [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return ViewModelBuilder<DeleteChannelDialogViewModel>.nonReactive(
+      viewModelBuilder: () => DeleteChannelDialogViewModel(),
+      builder: (context, vmodel, ch) {
+        return AlertDialog(
+          insetPadding: EdgeInsets.symmetric(horizontal: 65.0, vertical: 24.0),
+          scrollable: true,
+          title: Text('Delete Channel'),
+          // titlePadding: EdgeInsets.fromLTRB(24.0, 24.0, 24.0, 0),
```
Comment (jeremiahseun) [inline lib/ui/shared/dialogs/delete_channels_dialog.dart:22]: Please remove commented code when it's not been used

**Review by jeremiahseun — CHANGES_REQUESTED** [review]
Kindly make changes where required

**Pull request closed without merging**